src/helium_follower_service.erl

```diff
@@ -196,3 +191,22 @@ encode_follower_resp(TxnHash, Txn, TxnHeight, Timestamp) ->
 
 subscribed_type(_Type, []) -> true;
 subscribed_type(Type, FilterTypes) -> lists:member(Type, FilterTypes).
+
+validate_txn_filters(TxnFilters0) ->
+    case (catch lists:foldl(fun(BinType, AtomTypes) when is_binary(BinType) ->
+                    [binary_to_existing_atom(BinType, utf8) | AtomTypes];
```

Comment:
This could fail if the release is lazy-loading modules and that particular transaction module has not been called yet, no?

It might be safer to `try Module:module_info()` first, which would force the module load?

Reply:
Although I guess we did it that way before.